templates/home/home-en.html

```diff
@@ -132,34 +132,41 @@ <h3><a href="https://www.canada.ca/en/services/youth.html">Youth</a></h3>
 	</div>
 </section>
 
-<section class="home-your-gov gc-srvinfo well well-sm brdr-0">
+<section class="home-your-gov well well-bold well-sm brdr-0">
 	<div class="container">
 		<h2>Your government</h2>
-		<div class="row mrgn-bttm-md">
+		<div class="row mrgn-bttm-lg">
 			<div class="col-lg-7 col-xs-12 col-md-6">
-				<div class="wb-eqht row">
-					<div class="col-lg-6">
-						<h3><a href="https://www.canada.ca/en/contact.html">Contact us</a></h3>
-						<p>Department and agency contacts, change your address</p>
-					</div>
-					<div class="col-lg-6">
-						<h3><a href="https://www.canada.ca/en/government/dept.html">Departments and agencies</a></h3>
-						<p>A list of current government and related organizations</p>
-					</div>
-					<div class="col-lg-6">
-						<h3><a href="https://www.canada.ca/en/government/system.html">How government works</a></h3>
-						<p>Government structure, priorities, finances and regulations</p>
-					</div>
-					<div class="col-lg-6">
-						<h3><a href="https://open.canada.ca/en">Open government</a></h3>
-						<p>Accessible government, open information and data, open dialogue</p>
-					</div>
-				</div>
+				<ul class="row wb-eqht-grd list-unstyled small">
+					<li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm">
+						<a href="https://www.canada.ca/en/contact.html">Contact us</a>
```

Comment:
Suggestion, this following change is not required but more provided as FYI for the future component. 

The your gov HTML pattern can be simplified by defining style to the CSS "home-your-gov" and remove the need to use the following CSS modifier:
* `mrgn-bttm-lg`
* `list-unstyled`
* `small`
* `mrgn-tp-sm`
* `mrgn-bttm-sm`

Reply:
Thank you Pierre!